src/css/app.css

```diff
@@ -957,92 +1019,92 @@ a:hover {
   color: white;
 }
 
-.navbar .navbar-brand {
+.navbar.navigation-bar .navbar-brand {
   padding: 0;
 }
 
-.navbar .navbar-brand img {
+.navbar.navigation-bar .navbar-brand img {
   vertical-align: text-bottom;
 }
 
-.navbar .navbar-collapse form {
+.navbar.navigation-bar .navbar-collapse form {
   padding-top: 15px;
 }
 
-.navbar .navbar-collapse .navbar-nav .nav-item {
+.navbar.navigation-bar .navbar-collapse .navbar-nav .nav-item {
   padding-top: 15px;
 }
 
-.navbar .navbar-collapse .navbar-nav .nav-item:hover {
+.navbar.navigation-bar .navbar-collapse .navbar-nav .nav-item:hover {
   color: white;
 }
 
-.navbar .navbar-collapse .navbar-nav > a.nav-item:last-of-type {
+.navbar.navigation-bar .navbar-collapse .navbar-nav > a.nav-item:last-of-type {
   padding-bottom: 15px;
 }
 
-.navbar .static.navbar-nav {
+.navbar.navigation-bar .static.navbar-nav {
   flex-direction: row;
   margin-left: 30px;
 }
 
-.navbar .static.navbar-nav .nav-item {
+.navbar.navigation-bar .static.navbar-nav .nav-item {
   margin: -15px 0;
   padding: 15px 0;
   text-align: center;
 }
 
-.navbar .static.navbar-nav .nav-item {
+.navbar.navigation-bar .static.navbar-nav .nav-item {
   width: 60px;
 }
 
-.navbar .navbar-nav .nav-link {
+.navbar.navigation-bar .navbar-nav .nav-link {
   padding: 0;
 }
 
-.navbar .navbar-nav .nav-link img.add-listing {
+.navbar.navigation-bar .navbar-nav .nav-link img.add-listing {
```

Comment:
Just a comment that Josh is not a fan of namespacing. I realize that there is already a lot of it in this file because I was once following a convention from the website repo and haven't made it a priority to refactor all of it. But we should favor cascading and limit namespacing as much as possible.

Reply:
Thank @micahalcorn will convert it to cascading. I haven't done CSS in over 8 years so all these comments are super helpful.
I am happy to receive as many PR comments as possible. Since I want to get up to speed with the latest CSS best practices ASAP.